### ISSUE TITLE
Font too small in Windows

### ISSUE THREAD

**Issue by alensiljak** [Murmele/Gittyup#311]
Thanks again for the great tool. While using it, I have noticed that the small font size used in the interface is causing too much eye strain. I've compared, for example, the menu size to a few other apps, i.e.:

![image](https://user-images.githubusercontent.com/462445/195789797-867a64ed-9b48-4eb1-bde6-f7a23772d329.png)

Could this be increased to a comfortable size, please?
I see in the screenshots that in Linux the size seems quite fine so it might be that the font in Windows is a bit weird.

**Comment by exactly-one-kas** [comment]
![image](https://user-images.githubusercontent.com/16035731/195795790-1403c788-46ea-4a64-9774-8cacc7ff58bf.png)

I can't reproduce this with Windows 10 and Windows 11, all menu bars and font sizes are correct

You can however create a new theme from the current one and set a custom font size 
![image](https://user-images.githubusercontent.com/16035731/195796748-bec8df0b-3937-4a68-af4d-be73d983c62f.png)
![image](https://user-images.githubusercontent.com/16035731/195797322-8ceba342-3914-4ffc-81e1-103dfaffe001.png)

Does this work for you?

**Comment by alensiljak** [comment]
Thanks for the feedback! I see what's going on now. 
Since the laptop has a HD display, it is set to the recommended scaling of 125% (System > Display > Scale).
The other apps resize the fonts appropriately but Gittyup stays at the default value.

**Comment by alensiljak** [comment]
> Does this work for you?

As a workaround, yes, this also works. Thank you!

**Comment by kaushalyap** [comment]
I am experiencing the same but on Fedora.

**Comment by rokejulianlockhart** [comment]
@kaushalyap, that looks correct.

**Comment by kaushalyap** [comment]
> @kaushalyap, that looks correct. For comparison, what size should the text be?

Nope file menus and side bar fonts are small, font on those sections should be bit bigger at least to the size of font in center section.

**Comment by rokejulianlockhart** [comment]
@kaushalyap, you genuinely think that they should be as big as the center text...?

**Comment by kaushalyap** [comment]
> @kaushalyap, you genuinely think that they should be as big as the center text...?

Yes and then the center section should become bigger than that to make things correctly proportional.

**Comment by DKBurns** [comment]
Just to echo that on a 27" 4k monitor the text is really tiny in the commit history, recents, top panels too (at 125% scaling in W11). 

In Rstudio IDE, zooming _anywhere_ zooms the editor _and_ the side panels/buttons/etc, but not the menu bar at the top. This seems to work pretty well, and I wonder if similar behaviour might strike a nice balance here?

For now, I will make a custom theme which meets my requirements. I copied the dark theme and then added the below to the bottom of the file, then restarted. this seems to work (thank you for the instructions above - I just picked a random font for now to make it work):

```lua
-- font size 
theme["font"] = {
	family = "Arial",
	size = "18"
}

```

**Comment by JamesMowery** [comment]
Just gave this a try. I like the concept, but the interface is way too small. Borderline unusable. Tested on Arch and Ubuntu. Please add a global UI scaling option for those of us who are using HiDPI monitors. Thanks!

**Comment by HubKing** [comment]
It would probably better to open the `new theme.lua` file with the default text editor... or at least the add "Locate the file with the file manager" to the editor. Since the theme only applies when restarting the app, I have to keep re-open the options and switch to the Window tab and select "Edit current theme" again and again.

**Comment by HubKing** [comment]
> Just to echo that on a 27" 4k monitor the text is really tiny in the commit history, recents, top panels too (at 125% scaling in W11).

> ```lua
> -- font size 
> theme["font"] = {
> 	family = "Arial",
> 	size = "18"
> }
> ```

Is there any way to increase the item's height, too? I have set "20" for font, but it seems that item height doesn't grow enough. Lines look crammed.

![image](https://github.com/Murmele/Gittyup/assets/10900489/7105e448-e48d-47cd-8326-df110e4d0426)

**Comment by HubKing** [comment]
It seems that the font size is incorrect (too small) if installed from Flatpak, but correct if installed from AUR (Arch User Repository).

**Comment by RokeJulianLockhart** [comment]
> https://github.com/Murmele/Gittyup/issues/311#issuecomment-2099379295

@HubKing, might be best to file that at https://github.com/flathub/com.github.Murmele.Gittyup/issues/new then.

**Comment by RokeJulianLockhart** [comment]
> https://github.com/Murmele/Gittyup/issues/311#issuecomment-2099323937

@HubKing, indeed, they should adapt to the text size. https://forum.qt.io/post/270367 might be relevant here.

**Comment by thomaslevesque** [comment]
I can confirm the issue with 125% scaling. The menus and other UI elements are way too small.

![Image](https://github.com/user-attachments/assets/0bdedeed-498a-4d5f-8224-71a97f51e19c)

Gittyup uses Qt, right? Apparently [Qt has support for HiDPI displays](https://doc.qt.io/qt-6/highdpi.html).

**Comment by thomaslevesque** [comment]
> Gittyup uses Qt, right? Apparently [Qt has support for HiDPI displays](https://doc.qt.io/qt-6/highdpi.html).

Found another workaround on this page (for Windows). Add this to qt.conf in the bin folder:
```
[Platforms]
WindowsArguments = dpiawareness=1
```
(You might want to try different values, 0, 1 or 2. With 0 the text is a bit blurry for me. With 2, it's as small as before. 1 works best for me, text is the right size, and remains sharp. Sorry, I don't know what the different values mean!)

**Comment by AHSauge** [comment]
I suspect this got fixed with the upgrade to Qt6. I'm not able to reproduce this with latest master at least, which implies this likely is fixed in v2.0.0 already.
